.github/scripts/rebase_and_push.sh

```diff
@@ -0,0 +1,17 @@
+readonly GIT_HAS_NO_CHANGES=0 # 0 = clean, 1 = dirty
+
+git config user.name "$USER_NAME"
+git config user.email "$USER_EMAIL"
```

Comment:
Where are these shall variables getting set from?

Reply:
From the secrets, we already have these setup for `commit_and_push` & `amend_and_push`

Reply:
So if you look at actual GH Action job, we are passing these variables from there